web/src/components/Logout.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import firebase from "firebase/app";
+import "firebase/firestore";
```

Comment:
I don't think you are using firestore. Try removing and see if your code still works